node/sc/bridge_accounts.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package sc
+
+import (
+	"github.com/klaytn/klaytn/accounts"
+	"github.com/klaytn/klaytn/accounts/abi/bind"
+	"github.com/klaytn/klaytn/accounts/keystore"
+	"github.com/klaytn/klaytn/blockchain/types"
+	"github.com/klaytn/klaytn/cmd/homi/setup"
+	"github.com/klaytn/klaytn/common"
+	"io/ioutil"
+	"math/big"
+	"os"
+	"path"
+	"sync"
+)
+
+const (
+	GasLimit = 5000000
+)
+
+type accountInfo struct {
+	wallet   accounts.Wallet
+	address  common.Address
+	nonce    uint64
+	chainID  *big.Int
+	gasPrice *big.Int
+
+	isNonceSynced bool
+	mu            sync.RWMutex
+}
+
+// BridgeAccounts manages bridge account for main/service chain.
+type BridgeAccounts struct {
+	pAccount *accountInfo
+	cAccount *accountInfo
+}
+
+// NewBridgeAccounts returns bridgeAccounts created by main/service bridge account keys.
+func NewBridgeAccounts(dataDir string) (*BridgeAccounts, error) {
+	pWallet, pAccAddr, isLock, err := InitializeBridgeAccountKeystore(path.Join(dataDir, "parent_bridge_account"))
+	if err != nil {
+		return nil, err
+	}
+
+	if isLock {
+		logger.Warn("parent_bridge_account is locked. Please unlock the account manually for Service Chain")
+	}
+
+	cWallet, cAccAddr, isLock, err := InitializeBridgeAccountKeystore(path.Join(dataDir, "child_bridge_account"))
+	if err != nil {
+		return nil, err
+	}
+
+	if isLock {
+		logger.Warn("child_bridge_account is locked. Please unlock the account manually for Service Chain")
+	}
+
+	logger.Info("bridge account is loaded", "parent", pAccAddr.String(), "child", cAccAddr.String())
+
+	pAccInfo := &accountInfo{
+		wallet:   pWallet,
+		address:  pAccAddr,
+		nonce:    0,
+		chainID:  nil,
+		gasPrice: nil,
+	}
+
+	cAccInfo := &accountInfo{
+		wallet:   cWallet,
+		address:  cAccAddr,
+		nonce:    0,
+		chainID:  nil,
+		gasPrice: nil,
+	}
+
+	return &BridgeAccounts{
+		pAccount: pAccInfo,
+		cAccount: cAccInfo,
+	}, nil
+}
+
+// InitializeBridgeAccountKeystore initializes a keystore, imports existing keys, and tries to unlock the bridge account.
```

Comment:
How about adding a comment about its return value such as `It returns the first account of the keystore.`?